plugins/org.locationtech.udig.libs.netcdf/pom-libs.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.locationtech.udig</groupId>
+		<artifactId>pom-libs.parent</artifactId>
+		<version>0.0.2</version>
+		<relativePath>../../pom-libs.xml</relativePath>
+	</parent>
+
+	<!-- This script was created to automagically download the external libs 
+		required for uDig to build and export correctly. To download the required 
+		libraries, use: mvn package -f pom-libs.xml After they have downloaded, make 
+		sure you: - refresh the org.locationtech.udig.libs project in Eclipse; - 
+		verify the classpath (Manfiest Runtime tab) references the libs; - ensure 
+		the lib versions in the classpath are also correct. -->
+
+	<artifactId>pom-libs.download.geotools.netcdf</artifactId>
+	<name>uDig NetCDF Core libraries Download</name>
+
+	<packaging>pom</packaging>
+
+	<properties>
+		<!-- see geotools root pom for details -->
+		<geotools.version>22.1</geotools.version>
+		<imageio-ext.version>1.3.2</imageio-ext.version>
```

Comment:
move this version properties to root-pom-libs.xml to have single source to change in case of upgrading GeoTools or other dependencies